src/snapshot/svgtoimg.js

```diff
@@ -33,6 +33,12 @@ function svgToImg(opts) {
         }
 
         var canvas = opts.canvas;
+        var scale = opts.scale || 1;
+        var w0 = opts.width || 150;
+        var h0 = opts.height || 300;
```

Comment:
150 <-> 300 ?

Reply:
Good eyes! Done in https://github.com/plotly/plotly.js/pull/1979/commits/633882b8596fc620c122ff14389003e80808ae8f